src/value.rs

```diff
@@ -1,431 +1,51 @@
-// Copyright (c) 2017, 2022 ZettaScale Technology Inc.
-
+//
+// Copyright (c) 2024 ZettaScale Technology
```

Comment:
I think we don't need to update the year here.

Reply:
https://github.com/eclipse-zenoh/zenoh-python/pull/184#discussion_r1579621201